tools/cldr-code/src/main/java/org/unicode/cldr/util/DoctypeXmlStreamWrapper.java

```diff
@@ -84,12 +87,35 @@ private static String fixup(byte[] inbuf, String encoding) {
         }
     }
 
+    private static final Pattern numberAndType = PatternCache.get("^([0-9][0-9])\\/([a-zA-Z0-9]+)");
```

Comment:
we need to support any schema url including v45, etc.  Ideally we would check conformsTo here, but _at this level_ we don't typically have access to cldr-archive.